gcp-fsharp/Program.fs

```diff
@@ -5,9 +5,7 @@ open Pulumi.Gcp.Storage
 
 let infra () =
   // Create a GCP resource (Storage Bucket)
-  let bucket = Bucket("my-bucket",
-                    BucketArgs
-                        (Location = input  "US"))
+  let bucket = Bucket("my-bucket", BucketArgs(Location = "US"))
```

Comment:
I'm pretty sure `input` used to be required here... Has F# changed to allow implicit conversions? Or how does that work now?

Reply:
F# 6 has implicit conversions https://docs.microsoft.com/en-us/dotnet/fsharp/whats-new/fsharp-6#additional-implicit-upcast-conversions 

Though see comment above where an implicit conversion was used and compiled successfully but caused a runtime error. I want to look into that soon